ui/appui-react/src/appui-react/tools/ToolbarItems.tsx

```diff
@@ -0,0 +1,283 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Tools
+ */
+
+import { SelectionContextToolDefinitions } from "../selection/SelectionContextItemDef";
+import {
+  itemDefToToolbarActionItem,
+  itemDefToToolbarGroupItem,
+} from "../toolbar/ToolbarHelper";
+import type {
+  ToolbarActionItem,
+  ToolbarGroupItem,
+} from "../toolbar/ToolbarItem";
+import { CoreTools } from "./CoreToolDefinitions";
+
+/* eslint-disable deprecation/deprecation */
+
+/** @beta */
+export namespace ToolbarItems {
+  /** */
```

Comment:
Do we need to keep these descriptions if they're empty?

Reply:
Documentation added.